### DISCUSSION TITLE
borders reset

### DISCUSSION THREAD

**Discussion by ComplexGhost** [TownyAdvanced/Towny#6673] [Q&A] [answered]
after i changed the perms in towny to allow breaking and destroying, all the borders reset.

**Comment by LlmDl** [comment, marked as answer]
That is not something that happens when you change a town's permissions. I don't know what "all borders reset" means. 

Discussions is not the correct place to file a bug report. When you file a bug report do not skip the form and give proper explanations about what you have happen.
